flask_oidc_ext/__init__.py

```diff
@@ -224,14 +224,11 @@ def init_app(self, app):
         )
         assert isinstance(self.flow, OAuth2WebServerFlow)
 
-        # TODO use configurable salt string instead of hardcoded value to improve security
-        # create signers using the Flask secret key
-        self.extra_data_serializer = JSONWebSignatureSerializer(
-            app.config["SECRET_KEY"], salt="flask-oidc-extra-data"
-        )
-        self.cookie_serializer = JSONWebSignatureSerializer(
-            app.config["SECRET_KEY"], salt="flask-oidc-cookie"
-        )
+        # Other algorithms are available. We use the default that works with the app secret. 
+        self.signing_algorithm="HS256"
```

Comment:
Seeing that we only using this in the same dict each time, let's just create the dict here to clean it up a tiny bit.
```suggestion
        self.signing_algorithm={"alg": "HS256"}
```